packages/components/modal/src/Modal.mdx

```diff
@@ -3,10 +3,13 @@ title: 'Modal'
 type: 'component'
 status: 'stable'
 slug: /components/modal/
-github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Modal'
-storybook: 'https://f36-storybook.contentful.com/?path=/story/components-modal--default'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/modal/src'
```

Comment:
ah, sorry, it's here. hmmm, why we have 2 README files?

Reply:
we have three: `Modal`, `ModalConfirm` and `ModalLauncher`